executor/android/android_seccomp.h

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <assert.h>
+#include <linux/audit.h>
+#include <linux/seccomp.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <sys/prctl.h>
+#include <sys/syscall.h>
+
+#include <vector>
+
+// These headers are generated by the Android build system and need to be updated periodically.
+#include "func_to_syscall_nrs.h"
+
+#if GOARCH_arm || GOARCH_arm64
+
+#define DUAL_ARCH
```

Comment:
Do we need this DUAL_ARCH? Why? I suspect we don't need it. We always build for 1 target arch. We know during build if it's arm or arm64, 386 or amd64.

Reply:
lets leave it for now?

Reply:
I would drop it. We will never ever use it and it adds lots of code, functions, test scenarios and cognitive load in review.

Reply:
done